src/content/blog/node-test-migration.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: "Migrating 500+ tests from Mocha to Node.js"
+description: "A retrospective of how Astro migrated more than 500 test suites from Mocha to Node.js test runner."
+publishDate: "March 21, 2024"
+authors:
+  - ema
+  - bjorn
+lang: "en"
+coverImage: "/src/content/blog/_images/test-migration/og_astro_test_migration.webp"
+socialImage: "/src/content/blog/_images/test-migration/header_astro_test_migration.webp"
+---
+
+Over a month ago, we discussed a possible migration to the Node.js test runner. While we were sufficiently happy with Mocha, we are always looking to make our CI jobs faster.
+
+Relying on a test runner baked inside our runtime had some advantages for our main monorepo:
+- two fewer dependencies to install and maintain in our monorepo: `mocha` and `chai`;
+- maintainability, there are way more people involved in the Node.js project (hence the Node.js test runner);
+- we believe that the test runner will get better and better with the time, and eventually save some time in our CI workflows.
+
+## From an idea to a PoC
+
+The Astro monorepo has more than 500 testing suites: between integration tests and unit tests, we have 664 suites, with a total of 1603 tests. Tha majority of these tests are **integration** tests.
+
+An integration test, in our monorepo, means the creation of tiny Astro project, building a project with a specific environment - development, static generation (SSG) or dynamic generation (SSR), and run assertions over the **built** pages. That's right, each integration test requires `vite` to build and bundle the project.
+
+We didn't start the migration right away. Before taking a final decision, we wanted to make sure that moving away from Mocha was a mistake. Despite its quirks, Mocha is a **fine** test runner, it's been around for a long time, it is battle-tested. If you use Mocha, you are in good hands.
```

Comment:
```suggestion
An integration test, in our monorepo, means creating a tiny Astro project, building this project with a specific environment (development, static generation (SSG) or dynamic generation (SSR)), and then running assertions over the **built** pages. That's right, each integration test requires `vite` to build and bundle the project.

Before deciding to undertake this migration, we wanted to make sure that moving away from Mocha was not a mistake. Despite its quirks, Mocha is a very capable test runner! It's been around for a long time and is battle-tested. If you use Mocha, you are in good hands.
```
"Fine" can actually have negative connotations, especially when emphasized! (I guess it's *fine*...) so I suggested something that I think praises it the way you intended!